migrations/2019-04-18-233116_add_to_action_modifier_enum/up.sql

```diff
@@ -0,0 +1 @@
+ALTER TYPE action_modifier ADD VALUE 'own';-- Your SQL goes here
```

Comment:
Does this work for you? I get an error saying: `ALTER TYPE ... ADD cannot run inside a transaction block`

Reply:
honestly, I haven't figured out how to test it yet; I was going to ask you about it tomorrow. I'll look into it further and message you when it's ready for review, I still gotta figure out a few things first.

Reply:
Oh, you need to install diesel_cli somehow (I forgot). Then you just run `diesel migration run`. But yeah, sorry for preempting the review, I'll wait for your request next time.

Reply:
:D lol no worries man it's all good! The more feedback the better lol

Reply:
I'll figure out the migration and lyk

Reply:
@NicholasLYang ok I think it's ready!